index.test.js

```diff
@@ -0,0 +1,336 @@
+const { describe, expect, it } = require('@jest/globals')
+const emoji = require('.')
+
+describe('emojify', () => {
+  it('parses :emoji: in a string and replaces them with the right emoji', () => {
+    expect(
+      emoji.emojify(
+        'I :heart:  :coffee:! -  :hushed::star::heart_eyes:  ::: test : : :+1:+'
+      )
+    ).toBe('I ❤️  ☕! -  😯⭐😍  ::: test : : 👍+')
```

Comment:
I... think this is a bug in v2 that should be fixed?

```
Expected: "I ❤️  ☕! -  😯⭐😍  ::: test : : 👍+"
Received: "I ❤️  ☕! -  😯⭐😍  :: +1:+"Jest
```

Reply:
@omnidan thoughts? I'm not sure

Reply:
hmm yes it should be fixed, seems like a problem with the regex